oshdb-api/src/main/java/org/heigit/ohsome/oshdb/api/db/OSHDBIgnite.java

```diff
@@ -22,6 +22,13 @@
  * OSHDB database backend connector to a Ignite system.
  */
 public class OSHDBIgnite extends OSHDBDatabase implements AutoCloseable {
+  /**
+   * Specifies which algorithm to use when running OSHDB queries on Ignite.
+   *
+   * <p>Available computation modes: {@link #LOCAL_PEEK} (defalt) is good for small to medium
+   * sized areas, {@link #SCAN_QUERY} works best for large to global queries, and
+   * {@link #AFFINITY_CALL} is useful for streaming queries.
```

Comment:
little typo (default) and may be better us a list to explain those options.
```suggestion
   * <p>Available computation modes:
   * <ul> 
   *   <li>{@link #LOCAL_PEEK} (default) is good for small to medium sized areas 
   *   <li>{@link #SCAN_QUERY} works best for large to global queries 
   *   <li>{@link #AFFINITY_CALL} is useful for streaming queries
   * </ul>
```